modules/expo-bluesky-translate/index.ts

```diff
@@ -1,6 +1,66 @@
+import {Platform} from 'react-native'
+
+import ExpoBlueskyTranslate from './src/ExpoBlueskyTranslate'
+
+// can be something like "17.5.1", so just take the first two parts
+const version = String(Platform.Version).split('.').slice(0, 2).join('.')
+
+export const isAvailableIOS = Number(version) >= 17.4
+export const isAvailableAndroid = true
+
+// https://en.wikipedia.org/wiki/Translate_(Apple)#Languages
+const SUPPORTED_LANGUAGES = [
```

Comment:
kept this array since Google's MLKit solution supports all of these, so we use the lowest common denominator in this case (Apple's translation service)

Reply:
Just a nitpick, but I think it's better to have sorted alphabetically.

Reply:
Specifically opted to use Wikipedia’s order, which is by date added, so when they add new ones we can just add to the bottom.

Fwiw there’s no reason Android needs to limit itself to this!